docs/source/concepts/AbinsDataFormats.rst

```diff
@@ -0,0 +1,100 @@
+.. _Abins Data Formats:
```

Comment:
I can reach the `Abins Data Formats` page from both the Abins-v1 and Abins2D-v1 pages, but not from Concepts. It looks as if it needs to be added to `docs\source\concepts\index.rst`.

Reply:
It looks like most of the files are picked up automatically? I don't understand what is happening here.

```
.. Concepts master file
   It contains a hidden root toctree directive so that Sphinx
   has an internal index of all of the pages and doesn't
   produce a plethora of warnings about most documents not being in
   a toctree.
   See http://sphinx-doc.org/tutorial.html#defining-document-structure

.. _concepts contents:

==========
 Concepts
==========

.. toctree::
   :hidden:
   :glob:
   :maxdepth: 1

   *
   calibration/index
```

Reply:
Searching for another file in concepts, it doesn't seem to be explicitly listed anywhere in the source folder
```
~/s/m/d/source (docs-concepts-abins-data) [1]> rg -i abinsimplementation
concepts/AbinsImplementation.rst
1:.. _AbinsImplementation:

algorithms/Abins-v1.rst
49:If you are developing or modifying Abins, see the :ref:`AbinsImplementation` notes which outline some of the key conventions, practices and pitfalls.

algorithms/Abins2D-v1.rst
45:If you are developing or modifying Abins(2D), see the :ref:`AbinsImplementation` notes which outline some of the key conventions, practices and pitfalls.
```

Reply:
Yes, you are right. I will just try and compile all the docs again.